react/components/ProductImages/components/SelectedImage.js

```diff
@@ -1,23 +1,47 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
+import ImageZoom from './ImageZoom'
 
 import VTEXClasses from '../constants/productImagesClasses'
 
 /**
  * Selected Image component.
  * Display an image.
  */
-class SelectedImage extends Component {
+export default class SelectedImage extends Component {
+  constructor(props) {
```

Comment:
No need to user the constructor, you can change it for: 

state = {
    showZoom: false
}

Reply:
Done!